BitwardenShared/UI/Auth/CompleteRegistration/CompleteRegistrationView.swift

```diff
@@ -0,0 +1,175 @@
+import SwiftUI
+
+// MARK: - CompleteRegistrationView
+
+/// A view that allows the user to create an account.
+///
+struct CompleteRegistrationView: View {
+    // MARK: Properties
+
+    /// The store used to render the view.
+    @ObservedObject var store: Store<CompleteRegistrationState, CompleteRegistrationAction, CompleteRegistrationEffect>
+
+    // MARK: View
+
+    var body: some View {
+        VStack(spacing: 16) {
+            VStack(alignment: .leading, spacing: 0) {
+                Text(.init(store.state.headelineTextBoldEmail))
```

Comment:
The string has markup without the .init it is showing the markup. 
<img width="326" alt="image" src="https://github.com/user-attachments/assets/9ceabb90-5ba8-4a10-8287-19c3eb16062a">

Reply:
Oh, it's weird that it doesn't work directly. I think I would prefer to use `Text(LocalizedStringKey(store.state...))` to make it clear what's being initialized instead of the inferred init.

Reply:
Changed all .init to LocalizedStringKey